<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWiget</class>
 <widget class="QWidget" name="MainWiget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>721</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>551</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 224, 0)</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QTextBrowser" name="output_tb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>701</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>721</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>设置：</string>
     </property>
     <widget class="QTextEdit" name="CW_te">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>120</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>CW</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="CR_te">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="CV_pb">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>CR</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextEdit" name="DC_TE">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="lineWrapColumnOrWidth">
       <number>0</number>
      </property>
      <property name="placeholderText">
       <string>12,14</string>
      </property>
     </widget>
     <widget class="QPushButton" name="DC_PB">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CR_tb">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>50</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>VOLT_AC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="CW_pb">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>110</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 225, 225);</string>
      </property>
      <property name="text">
       <string>SET</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="CV_te">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>72</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>VOLT_DC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <zorder>label_9</zorder>
     <zorder>CV_pb</zorder>
     <zorder>label_10</zorder>
     <zorder>DC_PB</zorder>
     <zorder>CR_tb</zorder>
     <zorder>label_7</zorder>
     <zorder>CW_pb</zorder>
     <zorder>label_8</zorder>
     <zorder>CR_te</zorder>
     <zorder>CV_te</zorder>
     <zorder>CW_te</zorder>
     <zorder>DC_TE</zorder>
    </widget>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>通道1:</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>VOLT_DC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RATIO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="VOLT_DC_1_TB">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CUR_AC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CUR_DC_1_TB">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CUR_DC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CUR_AC_1_TB">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="RATIO_1_TB">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <zorder>label</zorder>
     <zorder>label_13</zorder>
     <zorder>label_14</zorder>
     <zorder>label_2</zorder>
     <zorder>CUR_DC_1_TB</zorder>
     <zorder>VOLT_DC_1_TB</zorder>
     <zorder>CUR_AC_1_TB</zorder>
     <zorder>RATIO_1_TB</zorder>
    </widget>
    <widget class="QPushButton" name="OFF_pb">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>360</y>
       <width>301</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(225, 225, 225);</string>
     </property>
     <property name="text">
      <string>断开</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="IP_Edit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>310</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;127.0.0.1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="ON_pb">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>360</y>
       <width>301</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(225, 225, 225);</string>
     </property>
     <property name="text">
      <string>连接</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>0</y>
       <width>361</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>通道2:</string>
     </property>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>VOLT_DC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>40</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RAIO</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="VOLT_DC_2_TB">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>100</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CUR_AC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CUR_DC_2_TB">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>72</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>CUR_DC</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTextBrowser" name="CUR_AC_2_TB">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>90</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextBrowser" name="RATIO_2_TB">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <zorder>label_5</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>label_6</zorder>
     <zorder>VOLT_DC_2_TB</zorder>
     <zorder>CUR_DC_2_TB</zorder>
     <zorder>RATIO_2_TB</zorder>
     <zorder>CUR_AC_2_TB</zorder>
    </widget>
    <widget class="QTextEdit" name="PORT_Edit">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>310</y>
       <width>191</width>
       <height>31</height>
      </rect>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'SimSun';&quot;&gt;12345&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>72</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>IP：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>320</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PORT：</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
